quickstart/examples/aggregation/README.md

```diff
@@ -0,0 +1,45 @@
+## Aggregation - Tumble window
```

Comment:
Let's create another nested folder inside `aggregation`  called `tumble window` and put this example there. In the future, we can add more examples for `tumble ` window or even `hop` window.

Reply:
done